lab-jaren/lib/parse-message.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+module.exports = exports = {};
+
+exports.nickCommand = function(message, socket) {
+  socket.username = message.split(' ').slice(1).join(' ').trim();
+  socket.write(`you are now ${socket.username}\n`);
+};
+
+exports.dmCommand = function(message, array) {
```

Comment:
Your /dm command should also include formatting to prepend the username before their message in the form of ``username: <message>``.